.pipelines/templates/template-prod-e2e-steps.yml

```diff
@@ -44,7 +44,6 @@ steps:
     docker run \
       --rm \
       --env CI=true \
-      --env RP_MODE=${{ parameters.rpMode }} \
```

Comment:
Will this not make old tests to fail? When we are running from "old binary" ?

Reply:
I don't think so: in PROD, the value is empty, so no-op, in INT, it's n/a because there is no "old".